### DISCUSSION TITLE
Starting / landing page should respect order of items in the admin settings

### DISCUSSION THREAD

**Discussion by MichlF** [mealie-recipes/mealie#946] [Feature Request]
**Before submitting this feature request I have**
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Within the settings of the admin panel, one can determine which categories should appear on the landing/startpage. However, the order of the listed category does not correspond to the order on the landing/page. 

**(Optional): Suggest A Solution**
Adaptation of the code such that the exact order is respected.

**Additional Information**
My Django skills are limited so I doubt I could write a useful PR at this point.
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened, so I'm closing it since it's no longer relevant.

I'm currently going through some old discussions, so apologies for the necro.
